tests/cli/test_ivcfmerge_batch_cli.py

```diff
@@ -24,11 +27,32 @@ def test_batch_sizes(input_paths_file, ref_merged_path, tmp_path, batch_size):
 
 
 @given(batch_size=st.integers())
-def test_temporary_dir(input_paths_file, ref_merged_path, tmp_path, batch_size):
+def test_temporary_dir(input_paths_file, ref_merged_path, tmp_path, tmpdir, batch_size):
     output_path = tmp_path / 'out.vcf'
+    temp_dir = tmpdir.mkdtemp()
 
     subprocess.run(['python3', 'ivcfmerge_batch.py', '--batch-size', str(batch_size),
-                    '--temp-dir', str(tmp_path), input_paths_file, str(output_path)])
+                    '--temp-dir', str(temp_dir), input_paths_file, str(output_path)])
 
     with output_path.open() as outfile, open(ref_merged_path, 'r') as ref_merged:
         assert outfile.read() == ref_merged.read()
+
+
+@given(batch_size=st.integers(min_value=2, max_value=5))
```

Comment:
Can we change the magic number 5 to `len(_input_paths())-1`? Same for other places.